apps/hash-api/src/graph/knowledge/primitive/entity.ts

```diff
@@ -10,6 +10,7 @@ import type {
   GraphResolveDepths,
   ModifyRelationshipOperation,
   PropertyConfidence,
+  UserEntityEditionProvenanceMetadata,
```

Comment:
I'm not sure about `User` / `user_defined`... in practice a lot of these fields will be provided by other machine processes. I appreciate the distinction between "Things the Graph sets" and "Things it is provided with", but I think `User` suggests too strongly that it is a human user rather than a caller of the Graph API (which might be some other server) that determines it.

Don't have any great alternative names... `Variable`, `External`... or possibly don't have a distinction in the API interface between `user_defined` and the createdX fields, and instead have a struct which excludes the Graph-determined fields for external input? 

We don't have to get this perfect at first since it's all the same thing in the db, but I'm not a fan of `User` here

Reply:
I agree that `User` is not great. What about `Provided` instead?

Reply:
`Provided` works, let's do it

Reply:
Done in [`9b2ca54` (#4304)](https://github.com/hashintel/hash/pull/4304/commits/9b2ca547e1c4c2c37c2a265343ffb60ba367f888)